ci/install_latest_cuda_toolkit.sh

```diff
@@ -2,17 +2,15 @@
 # Copyright (c) 2024, NVIDIA CORPORATION
 
 # Installs the latest CUDA Toolkit.
-# Supports CentOS 7 and Rocky Linux 8.
+# Supports Rocky Linux 8.
 
 yum update -y
 yum install -y epel-release
 
 OS_ID=$(. /etc/os-release; echo $ID)
-if [ "${OS_ID}" == "rocky" ]; then
-    yum install -y nvidia-driver
```

Comment:
Was this important? Is its removal the reason why CI is failing?

Reply:
I fixed this in 804afd4. We can build with driver stubs from `cuda-driver-devel-12-5` (to avoid installing a full driver in CI via `nvidia-driver`) but auditwheel complains about the driver stubs. I just told auditwheel to exclude `libcuda.so.1` and that's fine.